packages/server/lib/experiments.ts

```diff
@@ -68,9 +69,10 @@ const _summaries: StringValues = {
   ```
 */
 const _names: StringValues = {
-  experimentalFetchPolyfill: 'Fetch polyfill',
+  experimentalFetchPolyfill: 'Fetch Polyfill',
   experimentalInteractiveRunEvents: 'Interactive Mode Run Events',
-  experimentalSourceRewriting: 'Improved source rewriting',
+  experimentalSessionAndOrigin: 'Login Flows',
```

Comment:
Just checking, was it deliberate to still call this "Login Flows" here? I thought we changed the name, but I guess that discussion was only about the config option name, not the "experiment name" here.

Reply:
Yeah, I missed that during the rename. I have a PR out where I am updating it.
https://github.com/cypress-io/cypress/pull/21144/files#diff-31c253bb37cc2937da061c49cf2c161de03cebc7353a5b5fc6740ca5aaac1257

Reply:
I think this is being addressed in #21144 [here](https://github.com/cypress-io/cypress/pull/21144/files#r854471685)